karma.mc.config.js

```diff
@@ -18,11 +18,29 @@ const preprocessors = {};
 preprocessors[PATHS.testFilesPattern] = ["webpack"];
 
 module.exports = function(config) {
+  const isTDD = config.tdd;
   config.set({
-    singleRun: true,
+    singleRun: !isTDD,
     browsers: ["Firefox"],
     frameworks: ["mocha", "sinon", "chai"],
-    reporters: ["mocha"],
+    reporters: ["mocha", "coverage"],
+    coverageReporter: {
+      dir: "logs/system-addon",
```

Comment:
Sure, that's probably a good idea. No harm in putting it under `coverage/` just in case